src/components/Glossary/Glossary.js

```diff
@@ -22,17 +30,35 @@ class Glossary extends Component {
         return entries[index];
     }
 
+    getAcronym() {
+        return this.props.match.params.acronym;
+    }
+
+    isSearchEmpty = () => {
+        return this.getAcronym() === undefined;
+    }
+
     render() {
         const randomEntry = this.getRandomEntry();
+        const glossaryContainerClass = this.isSearchEmpty() ? "glossary__container--has-not-search" : "glossary__container--has-search";
         return (
-            <div className={"glossary__container"}>
-                <Link to={""} className={"glossary__logo"}>
-                    <img src={glossarioLogo} />
-                </Link>
-                <Search className={"glossary__search"}
-                        items={Object.keys(acronyms).sort()}
-                        handleSelect={this.handleAcronymChange}/>
-                <span className={"glossary__day-phrase"}>Você sabe o que é <Link className="emphasis pointer-hover light-accent lighter-hover" to={`/${randomEntry}`}>{ randomEntry }</Link>?</span>
+            <div className={"glossary__container " + glossaryContainerClass}>
+                <div className={"glossary__search-tools"}>
```

Comment:
ClassName n precisa do { } pra props, e o mesmo vale pra quaisquer props que são hardcoded strings... altera pra `className="..."`

Mesmo comentário vale pra repetições disso no código

Reply:
Acredito que essa alteração não seja necessária pois em todo o projeto aparecem hardcoded strings em volta de `{}`. Além disso, não há diferença de perfomance para que as `{}` sejam retiradas (https://stackoverflow.com/questions/39475986/react-is-there-a-different-between-using-curly-brackets-and-omitting-them). Se for necessario trocar, iria-se trocar no projeto todo ou só nesse arquivo?

Reply:
Concordo, vou dar resolve aqui e adicionar um comentário na issue do eslint